### PR TITLE
fix jira inline macro

### LINKED ISSUE #1411: Jira links substitution is not working after migrating from 2.1.0 to 3.3.1
**Describe the bug**
After migrating from v2.1.0 to v3.3.1 I noticed that the Jira link substitution stopped working.

**Expected behavior**
If we use for example `jira:ARC-102[]` in our adoc src file, when we run the task `generateHTML` the resulting HTML files should have the above part replaced with the Jira link pointing to our Jira server.

**Configuration**
 - docToolchain version: 3.3.1 Docker image

### PR DESCRIPTION
**Pull request by PacoVK**
closes #1411 

### All Submissions:

* [x] Did you update the `changelog.adoc`?
* [x] Does your PR affect the documentation?
* [x] If yes, did you update the documentation or create an issue for updating it?

### CODE REVIEW

File: core/src/main/groovy/org/docToolchain/scripts/asciidoc2confluence.groovy [hunk 1]
```diff
@@ -1,5 +1,10 @@
 package org.docToolchain.scripts
 
+/**
+ * THIS SCRIPT HAS BEEN DEPRECATED. IT IS NOT USED ANYMORE. PLEASE REFER TO THE NEW Asciidoc2ConfluenceTask
+ * IMPLEMENTATION INSTEAD. REFERENCE ONLY.
+ */
+
```
Comment (rdmueller) [inline core/src/main/groovy/org/docToolchain/scripts/asciidoc2confluence.groovy:7]: good to know 😁

**Review by rdmueller — APPROVED** [review]
LGTM

**Comment by rdmueller** [comment]
wow. that was quite a change!

**Comment by PacoVK** [comment]
@rdmueller following the boy scout rule the change was bigger than it would have been to fix only, but the docs were also incomplete and i think i reduced a bit of complexity removing at least one ambigious config parameter that was only used in that area :)

**Pull request merged**